test/test_transforms.py

```diff
@@ -8027,6 +8028,14 @@ def test_transform_rb(self, rbclass):
     def test_transform_inverse(self):
         raise pytest.skip("No inverse for KLRewardTransform")
 
+class TestPermuteTransform:
+    @pytest.mark.parametrize("batch", [[], [2], [2, 4]])
+    def test_transform_no_env(self, batch):
+        D, W, H, C= 8, 32, 64, 3
+        trans=PermuteTransform(dims=(-1, -4,-2, -3), in_keys=["pixels"],) # DxWxHxC => CxDxHxW
+        td=TensorDict({"pixels": torch.randn((*batch, D, W, H, C))}, batch_size=batch)
+        td=trans(td)
+        assert td["pixels"].shape == torch.Size((*batch, C, D, H, W))
```

Comment:
Better to use the TransformBase template to make sure that everything works properly. 
You will need an env to check that the env specs match what is expected.
We can use a conv mocking env perhaps (`DiscreteActionConvMockEnv`)?
Feel free to skip the tests that are not relevant to that class.

Reply:
I've committed new testing codes with TransformBase template. And it's unable to run test since the initialization of torchrl (import torchrl) is failed for the sources on branch main, with error message:  "ModuleNotFoundError: No module named 'torchrl._torchrl'"